isso/db/__init__.py

```diff
@@ -121,3 +121,39 @@ def first(rv):
 
                 con.execute('PRAGMA user_version = 3')
                 logger.info("%i rows changed", con.total_changes)
+
+        # "text" field in "comments" became NOT NULL
+        if self.version == 3:
+            with sqlite3.connect(self.path) as con:
+                con.execute("BEGIN TRANSACTION")
```

Comment:
I like this wrapping in a `TRANSACTION`, it's good practice. Might make sense to do this for the other migration steps as well.